README.md

```diff
@@ -72,9 +72,9 @@ If the feature is enabled then the _activeComponent_ will render else it renders
 
 Feature takes these **props**
 
-* name = ""
-* inactiveComponent = noop
```

Comment:
I've noticed that everytime you change something in the readme, your prettier version or editor changes all of these bullet points from `*` to `-`. It shouldn't be doing that, it's creating a bunch of unnecessary changes. Maybe deleting your node modules folder and re-running npm install will fix this if its prettier version related. If it's your editor maybe you can debug it.